imageio/plugins/feisem.py

```diff
@@ -0,0 +1,72 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2016, imageio contributors
+# imageio is distributed under the terms of the (new) BSD License.
+
+from __future__ import absolute_import, unicode_literals
+
+from .tifffile import TiffFormat
+
+from .. import formats
+
+
+class FEISEMFormat(TiffFormat):
+    """Provide read support for TIFFs produced by an FEI SEM microscope."""
```

Comment:
This is the docstring that ends up [here](http://imageio.readthedocs.io/en/latest/formats.html). I think it could mention that the format derives from the Tifffile format, and that it supports the same parameters. Further, this would be the place to document the kwargs in `get_data`. See the avbin plugin for an example.